### DISCUSSION TITLE
Flectra On-Premise Installation Under Docker

### DISCUSSION THREAD

**Discussion by jalbright1957** [flectra-hq/flectra#38] [Q&A] [answered]
I have a Ubuntu 22.04 server with Docker installed that I just put together a couple days ago.  I have tried three different times to install Flectra using docker pull flectrahq/flectra:latest fround at https://hub.docker.com/r/flectrahq/flectra/tags & then after the pull has been completed, following the instructions on the opposite tab at https://hub.docker.com/r/flectrahq/flectra.  Each time I start Flectra using "docker start -a flectra", I can see everythig running on the display but am unable to connect using a browser as it states it cannot connect to the database.   I am only assuming that performing the pull request installs the Polstgresql database & creates the user & I can verify it is there by displaying the Postgres container.  What is the proper procedure to get Flectra installed & proper method to connect?  My server is on our home network so connecting to Flectra I use the ip address of the server & port number  192.167.1.20:7073.  Is there anyone that can shed some light on this?

**Comment by parthivgls** [comment, marked as answer]
you can refer to "How To" instructions from here : https://hub.docker.com/r/flectrahq/flectra
